### PR TITLE
Chart cosmetics

### PR DESCRIPTION
**Pull request by pepov**
- chore: remove generated dependency charts from logging-demo
- chore: add tag for doc

### CODE REVIEW

File: charts/log-generator/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: log-generator
-version: 0.5.1
+version: 0.5.2
 appVersion: v0.5.0
```
Comment (genofire) [inline charts/log-generator/Chart.yaml:5]:
```suggestion
appVersion: 0.5.0
```

**Review by genofire — CHANGES_REQUESTED** [review]
appVersion should be only the version without tag

```
          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default (printf "v%s" .Chart.AppVersion) }}"
```

File: charts/log-generator/Chart.yaml (hunk 1 above)
Comment (pepov) [reply charts/log-generator/Chart.yaml:5]: Why?

From https://helm.sh/docs/topics/charts/
> appVersion: The version of the app that this contains (optional). Needn't be SemVer. Quotes recommended.

Comment (genofire) [reply charts/log-generator/Chart.yaml:5]: same format - but okay

**Review by genofire — APPROVED** [review]

**Pull request merged**